{
  "schema_version": "1.4.0",
  "id": "GHSA-q5x2-7c2j-3rwq",
  "modified": "2022-05-13T01:38:32Z",
  "published": "2022-05-13T01:38:32Z",
  "aliases": [
    "CVE-2016-9468"
  ],
  "details": "Nextcloud Server before 9.0.54 and 10.0.1 & ownCloud Server before 9.0.6 and 9.1.2 suffer from content spoofing in the dav app. The exception message displayed on the DAV endpoints contained partially user-controllable input leading to a potential misrepresentation of information.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2016-9468"
    },
    {
      "type": "WEB",
      "url": "https://github.com/nextcloud/server/commit/7350e13113c8ed484727a5c25331ec11d4d59f5f"
    },
    {
      "type": "WEB",
      "url": "https://github.com/nextcloud/server/commit/a4cfb3ddc1f4cdb585e05c0e9b2f8e52a0e2ee3e"
    },
    {
      "type": "WEB",
      "url": "https://github.com/owncloud/core/commit/96b8afe48570bc70088ccd8f897e9d71997d336e"
    },
    {
      "type": "WEB",
      "url": "https://github.com/owncloud/core/commit/bcc6c39ad8c22a00323a114e9c1a0a834983fb35"
    },
    {
      "type": "WEB",
      "url": "https://hackerone.com/reports/149798"
    },
    {
      "type": "WEB",
      "url": "https://nextcloud.com/security/advisory/?id=nc-sa-2016-011"
    },
    {
      "type": "WEB",
      "url": "https://owncloud.org/security/advisory/?id=oc-sa-2016-021"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-284",
      "CWE-451"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2017-03-28T02:59:00Z"
  }
}